WallsAndHoles/editor.cpp

```diff
@@ -100,12 +100,11 @@ Editor::~Editor()
 void Editor::newMap()
 {
     NewMapDialog nmd;
-    nmd.exec();
 
-    if(nmd.accepted){
-        setTileMap(new TileMap(QSize(nmd.result.width, nmd.result.height), nmd.TileMapProperty));
-    }else{
```

Comment:
In this case just let control run out (don't need return if there is nothing after it)